src/main/java/com/meilisearch/sdk/KeysHandler.java

```diff
@@ -77,7 +89,6 @@ Key updateKey(String key, Key options) throws Exception {
      * @throws MeilisearchException if client request causes an error
      */
     void deleteKey(String key) throws MeilisearchException {
-        String urlPath = "/keys/" + key;
-        httpClient.delete(urlPath, String.class);
+        httpClient.delete(new KeysQuery().toQuery(key), String.class);
```

Comment:
Same as for `getIndex` and `getIndexes`, should `deleteKey` and `deleteIndex` somehow work in similar ways?

Reply:
Same answer as for `getKey()`